coralproject/talk
# Add flag option

## What does this PR do?

Adds an option to explain a flag (Pivotal story https://www.pivotaltracker.com/n/projects/1863625/stories/135985387)

## How do I test this PR?

1. Report a comment.
2. Click any reason for the report.
3. You should see a text field asking you to provide an optional reason for the flag.
4. Enter a reason.
5. This reason should be sent to the server and stored in the database.
